src/protagonist/API/Features/Queues/CustomerQueueController.cs

```diff
@@ -106,13 +107,24 @@ public async Task<IActionResult> GetCustomerQueue([FromRoute] int customerId, Ca
                 }
             }
         }
-        
+
+        if (images.Members != null)
+        {
+            // ReSharper disable once ReturnValueOfPureMethodIsNotUsed
+            images.Members.Select(c =>

```

Comment:
`.Select()` with side effects seems odd, is it better to filter those with `string.IsNullOrEmpty(id)` and only update those?